packages/kit/src/runtime/client/renderer.js

```diff
@@ -384,6 +384,16 @@ export class Renderer {
 					deep_linked.scrollIntoView();
 				} else {
 					scrollTo(0, 0);
+					if (document.body.scrollTop > 0) document.body.scrollTo(0, 0);
```

Comment:
can you add a comment explaining how this line is different than the preceding one?